main.tf

```diff
@@ -4,7 +4,11 @@ terraform {
 
 locals {
   master_instance_name = (
-    var.full_name_master_instance == "" ? format("mysql-%s-%s", var.name_master_instance, var.name_suffix) : var.full_name_master_instance
+    var.full_name_master_instance != ""
+    ?
+    format("%s-%s", var.full_name_master_instance, var.name_suffix)
+    :
+    format("mysql-%s-%s", var.name_master_instance, var.name_suffix)
```

Comment:
We already have some legacy mysql instances in GCP which were created before IAC and they don't have these name suffix patterns. The `var.full_name_master_instance` was created to be able to import those instances into IAC.

With this new change, those instances will undergo a destructive change. Today, if we want to be able to name our instances differently, we can leverage on this `full_name_master_instance` variable to override the full name